examples/Mediawiki/ProvideAlice.yaml

```diff
@@ -0,0 +1,96 @@
+fixture: Alice
```

Comment:
Do the fixture creations also serve as tests?

Reply:
Fixture setups can apply assertions to responses, just like a suit's setup sequence can. When they fail, all suites that depend on them should be skipped. When a test fails, that test gets marked as failed, but it doesn't have any consequences for other tests, not even within the same suite.

So, fixtures act as globally shared setup sequences. They are logically not tests. But there is no functional difference between an interaction in a setup, a fixture, or a test. The only difference is in the interpretation of failures.

Reply:
In more practical terms: a fixture guarantees a certain known state to exist. It may contain assertions that check that this is the case. This could also be (ab)used for testing, but that would defy the purpose of having fixtures in the first place.